### DISCUSSION TITLE
Can we use Panda css with Lit ?

### DISCUSSION THREAD

**Discussion by FatmaKahloun** [lit/lit#4638] [Q&A]
I want to know if I can use Panda CSS as my css-in-js library with Lit to build my web components, and if it's possible can you please give me an example of how can I use it ? 
I had try so many time to use panda css with lit but I can't do it and I'm really need to get the response if it's possible or not and how.
And I thank you so much for your response.

**Comment by sorvell** [comment]
I'm sure you can with some nitty gritty setup, but it doesn't appear Panda has a Lit integration out of the box and because, like tailwind, it appears to require integration with the templating system to "discover" and tree shake css, it can be some work to setup and integrate. This is one of the trade offs with any build-centric css framework.

A couple alternatives to consider (yes, these are not strictly panda.css but provide some similar features):
* [twind](https://twind.style/): [sample](https://lit.dev/playground/#gist=931b39fc0b59915336b038aa62caae8a)
* [relevant lit rrfc](https://github.com/lit/rfcs/issues/38): [sample](https://lit.dev/playground/#gist=794e1080ed9d281d8655853ff566a8f6) shows an inline `rule` directive and a `createVariants` helper inspired by panda.
